Lib/ruby/director.swg

```diff
@@ -151,6 +151,8 @@ namespace Swig {
     const char *what() const throw() {
       return swig_msg.c_str();
     }
+
+    int status;
```

Comment:
Hi,
calling `rb_jump_tag(status)` is a standard way to re-raise an exception raised while`rb_protect` called. Please see [reference](https://docs.ruby-lang.org/en/2.4.0/extension_rdoc.html#label-Control+Structure).  I am not sure calling `rb_raise(error)` or `rb_exc_raise(error)` always works. For that reason, I have added the public member variable.